supabase/migrations/20260811134026_fix_rbac_jwt_priority_over_capgkey.sql

```
@@ -0,0 +1,339 @@
+  RETURN v_allowed;
+END;
+$$;
+
+COMMENT ON FUNCTION public.rbac_check_permission_direct(
+  text,
+  uuid,
+  uuid,
+  character varying,
+  bigint,
+  text
+) IS 'Direct RBAC permission check. Non-empty p_apikey selects the API-key principal; otherwise the user principal is used. Applies channel overrides and password/2FA gates for user checks. JWT-vs-capgkey selection for RLS is handled by rbac_check_permission_request.';
```

SQLFluff (4.2.2)

[error] Line is too long (269 > 80).

(LT05)